bee-protocol/src/worker/transaction/transaction.rs

```diff
@@ -186,7 +196,7 @@ mod tests {
         let protocol_config = ProtocolConfig::build().finish();
         block_on(Protocol::init(protocol_config, network));
 
-        assert_eq!(tangle().size(), 0);
+        assert_eq!(tangle().inner.size(), 0);
```

Comment:
Should we have a forward for this ?

Reply:
In case it wasn't clear, I was speaking of a `size` forward :)

Reply:
yea, I think in this case it's very well justified! Will add it.